lib/ndr_import/file/xml.rb

```diff
@@ -16,32 +16,87 @@ def initialize(*)
         super
 
         @pattern_match_xpath = @options['pattern_match_record_xpath']
+        @xml_file_metadata   = @options['xml_file_metadata']
+        @options['slurp'] ? prepare_slurped_file : prepare_streamed_file
       end
 
       private
 
+      def prepare_slurped_file
+        @doc = read_xml_file(@filename)
+        slurp_metadata_values
+      end
+
+      def prepare_streamed_file
+        with_encoding_check(@filename) do |stream, encoding|
+          @stream   = stream
+          @encosing = encoding
```

Comment:
Replace `@encosing` -> `@encoding`.

If this is important, do we need more test coverage?

Reply:
Thanks @bshand - 3185887

I _think_ the tests in `XmlStreamingTest` should cover the encoding logic, but agree this bug should've been found via a test 🤔